easybuild/easyconfigs/q/Qwt/Qwt-6.1.5-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'Qwt'
+version = '6.1.5'
+
+homepage = 'http://qwt.sourceforge.net/'
+description = """The Qwt library contains GUI Components and utility classes which are primarily useful for programs
+ with a technical background."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['4076de63ec2b5e84379ddfebf27c7b29b8dc9074f3db7e2ca61d11a1d8adc041']
+
+builddependencies = [
+    ('binutils', '2.34'),
+]
+
+dependencies = [
+    ('Qt5', '5.14.1'),
+]
+
+skipsteps = ['configure']
+
+prebuildopts = "sed -i 's@QWT_INSTALL_PREFIX[ ]*=.*@QWT_INSTALL_PREFIX = %(installdir)s@g' qwtconfig.pri && "
```

Comment:
Qwt-6.1.4-GCCcore-8.2.0.eb and *-8.3.0.eb have `export QWT_PREFIX=%(installdir)s && ` instead of this whole line. I think the `sed` based solution might be outdated (?). Just exporting the `QWT_PREFIX` is definely more elegant, so if that works, I would personally prefer that

Reply:
This need the Qwt-6.1.4_Fix_prefix_and_pkgconfig.patch. I will add it into next commit.

Reply:
Resolved in https://github.com/easybuilders/easybuild-easyconfigs/pull/11064/commits/ae8e720b1f4a8af39096f0216294be2d416d7aef

using Qwt-6.1.4 patch